### PR TITLE
fix(i18Config) : add language detection from localstorage for i18n

### PR DESCRIPTION
**Pull request by Mehdi-c**
# بسم الله الرحمن الرحيم

## What's new in this PR?

Add user selected language in localstorage 

لله الحمد والمنة

### CODE REVIEW

**Comment by zerubeus** [comment]
LGTM, Awesome, barakallahu fik for the great effort

**Review by zerubeus — APPROVED** [review]

**Comment by zerubeus** [comment]
Screenshot 2020-04-26 at 11 35 58

@Mehdi-c the build fail, I think it is because he can't figure out what window is in the server side maybe you can try something like this I'm not sure: 

```js
export const isBrowser = () => typeof window !== "undefined"
```

File: src/i18n/config.ts [hunk 1]
```diff
@@ -17,7 +39,7 @@ i18n
         translations: require('../locales/fr.json'),
       },
     },
-    lng: 'ar',
+    lng: window.localStorage[localStorageName] || 'ar',
```
Comment (zerubeus) [inline src/i18n/config.ts]: here

**Review by zerubeus — CHANGES_REQUESTED** [review]

File: src/i18n/config.ts (hunk 1 above)
Comment (Mehdi-c) [reply src/i18n/config.ts]: arg, i will work on it

Comment (zerubeus) [reply src/i18n/config.ts]: barakallahu fik

Comment (Mehdi-c) [reply src/i18n/config.ts]: This branch was successfully deployed
1 active deployment

**Comment by zerubeus** [comment]
barakallhu fik @Mehdi-c

**Pull request merged**